src/mvs/mvsNodeActions.ts

```diff
@@ -33,9 +33,19 @@ export async function uploadDialog(node: ZoweNode, datasetProvider: DatasetTree)
     datasetProvider.refresh();
 }
 
+export function getDatasetLabel(node: ZoweNode) {
+    if (node.mParent && node.mParent.contextValue === "favorite") {
+        const profileEnd = "]: ";
+        const profileIndex = node.label.indexOf(profileEnd);
+        return node.label.substr(profileIndex + profileEnd.length, node.label.length);
+    }
+    return node.label;
+}
```

Comment:
We do need to refactor this code at some point to store dataset name and profile name as separate attributes on the node object instead of concatenating the strings.

Reply:
That's a good observation.
Is there an issue open about it?

Reply:
No, but I can create one.

Reply:
https://github.com/zowe/vscode-extension-for-zowe/issues/136